### ISSUE TITLE
Fermata Auto app not showing up in Oneplus Nord CE 2

### ISSUE THREAD

**Issue by srikanth2042** [AndreyPavlenko/Fermata#201]
Fermata auto app is not showing up in car’s screen once connected. I see fermata control app but there are no options to view YouTube.
Android version 12
Android auto version 8.0.623354-release
Developer mode is turned on and Unknown sources are allowed too in Android auto settings

**Comment by fcaronte** [comment]
yes is a know thing, unfortunately all custom android auto app won't work on any Oppo/Realme/OnePlus smartphone and we can't understand why. probably some firmware part give a bug and make it not to work with the usually trick

**Comment by srikanth2042** [comment]
It might be because oneplus phones run on oxygen os which is a layer on top of android os?

**Comment by Hirosashii** [comment]
> It might be because oneplus phones run on oxygen os which is a layer on top of android os?

If it helps at all, I have a OnePlus 7 Pro, and Fermata Auto _did_ work on it while I was OxygenOS version OP7Pro_02_BETA_5. It was also Android Auto 8.2.623913-release. I have since updated to Android 12 and now I have the same situation as srikanth2042. But I just wanted to make it known that as a OnePlus user, I can confirm that it did indeed work before under certain circumstances.

**Comment by Hirosashii** [comment]
I actually just downgraded to 11 to see if it would work again. It was a pain to try to get OP7Pro_02_BETA_5 working, so I ended up with **Oxygen OS 11.0.5.1.GM21AA** and it looks like Fermata Auto still works even here. It's just when I updated to 12 that it had an issue.

https://i.postimg.cc/D0zYcnPJ/Untitled.png
https://i.postimg.cc/QMr4ypqr/Screenshot-20221102-164324.jpg
https://i.postimg.cc/XqXQCxn9/IMG-20221102-165245.jpg

**Comment by srikanth2042** [comment]
@Hirosashii yes my oneplus phone is also running on Android 12, fermata auto is not working either. So the issue is with android 12 only.

**Comment by fcaronte** [comment]
> @Hirosashii yes my oneplus phone is also running on Android 12, fermata auto is not working either. So the issue is with android 12 only.

No fermata work on Android 12 and 13 too, the problem are Oppo based firmware like realme, OnePlus and Oppo phone we don't know why they aren't work

**Comment by AndreyPavlenko** [comment]
Try this - https://github.com/AndreyPavlenko/Fermata/discussions/122#discussioncomment-4307775
